src/sentry/conf/server.py

```diff
@@ -1132,7 +1139,7 @@ def create_partitioned_queues(name):
 )
 
 # Delay (in ms) to induce on API responses
-SENTRY_API_RESPONSE_DELAY = 0
+SENTRY_API_RESPONSE_DELAY = 150 if IS_DEV else None
```

Comment:
@mattrobenolt then we have to maintain that configuration *somehow*. We can remove/override/ensure the delay doesnt exist in tests, but for development its relevant since it replicates real-world conditions. I run with a 1s delay for example.